src/components/profile-view/profile-view.jsx

```diff
@@ -0,0 +1,122 @@
+import { React, useEffect, useState } from "react";
+import MovieCard from "../movie-card/movie-card";
+import { Link } from "react-router-dom";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import FavoriteMovies from "./fav-movies";
+import moment from "moment/moment";
+
+const ProfileView = ({ user, token, setUser, movies }) => {
+  console.log(user, movies)
```

Comment:
To keep your clean, please remove any console.logs that were just for debugging...